config/environments/production.rb

```diff
@@ -24,7 +24,11 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
Right, I didn't remember this was when an asset is not found (if `false` you raise an error, if `true` you pass through the pipeline). I'd say it's ok as it is. Thank you!